browser_history/browsers.py

```diff
@@ -124,38 +110,16 @@ class Edge(Browser):
 
 class Opera(Browser):
     """Opera Browser
-
-    Supported platforms (TODO: Mac OS support)
-
-    * Linux, Windows
-
-    Profile support: No
+    Supported platforms (TODO: Windows support)
+    * Mac OS
+    Profile support: Yes
     """
     name = "Opera"
+    mac_path = 'Library/Application Support/com.operasoftware.Opera/'
 
-    linux_path = '.config/opera'
-    windows_path = 'AppData/Roaming/Opera Software/Opera Stable'
-
-    profile_support = False
+    profile_support = True
+    profile_dir_prefixes = Chrome.profile_dir_prefixes
 
     history_file = Chrome.history_file
 
     history_SQL = Chrome.history_SQL
-
-class OperaGX(Browser):
-    """Opera GX Browser
-
-    Supported platforms
-
-    * Windows
-
-    Profile support: No
-    """
-    name = 'OperaGX'
-
-    windows_path = 'AppData/Roaming/Opera Software/Opera GX Stable'
-
-    profile_support = False
-
-    history_file = Chrome.history_file
-    history_SQL = Chrome.history_SQL
```

Comment:
You accidentally reverted a feature added by another PR :sweat_smile:

Reply:
Alright 
Will make a new PR then